include/ccf/crypto/pem.h

```diff
@@ -134,6 +134,23 @@ namespace crypto
     return "Pem";
   }
 
+  static std::vector<crypto::Pem> split_x509_cert_bundle(
```

Comment:
I think we could do this with `nonstd::split` (defined in `ds/nonstd.h`)? Would then need an additional parsing of each as `Pem`s. No urgent though - makes sense to leave this until the unit test in #4142, and simplify the implementation if it still passes that test.

Reply:
Added a note to https://github.com/microsoft/CCF/issues/4142